doc/source/concept_and_workflow/systemdeps.rst

```diff
@@ -0,0 +1,96 @@
+.. _systemdeps:
+
+Host Requirements To Build Images
+---------------------------------
+
+When building OS images, several tools and sub-systems are used
+and required on the host {kiwi} is called at. For example to
+build a virtual disk image tools to create a partition table
+and also tools to create filesystems needs to be present on the
+host that builds the image.
+
+The number of required components depends on the selected image
+type and the *features* used with the image. We cannot expect
+the users of {kiwi} to know about each and every component that
+is needed to build the image. Therefore a concept to help with
+the host requirements exists and is named `kiwi-systemdeps`
+
+The `kiwi-systemdeps` concept consists out of a collection of
+sub-packages provided with the `python3-kiwi` main package. Each
```

Comment:
nit: I'd say the main package should be `python-kiwi` as this is how src package is called.